lib/vm/src/lib.rs

```diff
@@ -62,6 +60,11 @@ pub use crate::vmcontext::{
 pub use crate::vmoffsets::{TargetSharedSignatureIndex, VMOffsets};
 use loupe::MemoryUsage;
 pub use wasmer_types::VMExternRef;
+#[deprecated(
+    since = "2.1.0",
+    note = "ModuleInfo, ExportsIterator, ImportsIterator should be imported from wasmer_types."
+)]
```

Comment:
I'm not sure why we should make it deprecated. It's fine to re-export types from another crate here. What do you think?